### ISSUE TITLE
Add support for running WASM binaries

### ISSUE THREAD

**Issue by prabirshrestha** [holzschu/a-shell#28]
https://github.com/wasm3/wasm3

**Comment by holzschu** [comment]
Yes, I am working on this. I would rather use Apple's own WASM JIT compiler, which should be more efficient (and compact). If this fails, I'll fall back to wasm3.

In the current version, the wasm command is already there. Of course, without wasi, it is practically useless.

**Comment by holzschu** [comment]
I am going to have the extreme pleasure to mark this issue as "fix committed" (and actually released by Apple in the AppStore)

**Comment by prabirshrestha** [comment]
Awesome.
